external-import/microsoft-sentinel-incidents/src/microsoft_sentinel_incidents_connector/utils.py

```diff
@@ -23,6 +23,9 @@ def format_datetime(date_str: str | None) -> str:
     :return: The formatted date string in ISO 8601 format with 'Z' as the timezone indicator.
     """
     if date_str is not None and date_str.strip():
+        # some 'CreatedTimeUtc' date doesn't contains 'Z' (see malware evidence)
+        if not date_str.endswith("Z"):
```

Comment:
@romain-filigran Can you please add a security here (never know what MS is doing) with and not endswith +00:00 (see replace just after)